src/components/Card/Card.mdx

```diff
@@ -208,6 +209,15 @@ This example can be found in the [Turbine homepage](https://nypl-ds-test-app.ver
 
 <Canvas of={CardStories.CardFullClickTurbineExample} />
 
+## Card With Full-Click Functionality and Tooltip
+
+When the `Card` is fully clickable, an internal component wraps everything in a link, which in turn overrides the pointer events that trigger a `Tooltip`.
+
+If you need a tooltip within a clickable `Card`, wrap the entire `Card` component inside the `Tooltip` and use the `placement` and `offset` props to move the `Tooltip` to display next to the intended reference.
+Keep in mind that `offset` is static pixel values, so check where the `Tooltip` displays on all breakpoints. You may need to keep track of the offset values through state, and update them conditionally with the `useNYPLBreakpoints` hook.
```

Comment:
Good catch. I was not resizing and checking every time because I don't think the use-case of DC needs it for for widths smaller than desktop and the image is always the same size.

I'm concern about dynamically keeping track of this value because, if it's used on a site like DC with various tooltips, then the page might get laggy from all the calculations. The height of the image can probably be used to dynamically generate the offset. Otherwise, I'm thinking we don't advertise this as a feature because not getting the exact location of the tooltip on a responsive Card can be tricky and expensive. 

@7emansell try it out and let us know how it goes.